### PR TITLE
App strings reworked 10

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/resources/i18n/translations/en.json
Comment (comradekingu) [inline src/resources/i18n/translations/en.json:607]: Not sure

File: src/resources/i18n/translations/en.json
Comment (mitchdowney) [inline src/resources/i18n/translations/en.json:717]: It's fine if you want to italicize these, but this won't render correctly in the React Native Text component. Making this change would need it's own Github issue. I'll remove these after merging.

File: src/resources/i18n/translations/en.json
Comment (mitchdowney) [inline src/resources/i18n/translations/en.json:745]: I was of the impression that we have to be explicit about when a user is "Leaving Podverse" for app store compliance. I'll merge this though and we'll see if they flag us.

**Review by mitchdowney — APPROVED** [review]

File: src/resources/i18n/translations/en.json
Comment (comradekingu) [reply src/resources/i18n/translations/en.json:745]: When I click the GitHub link in "About" I am not actually leaving Podverse.
It just opens a web browser.

**Pull request merged**
